xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodec.h

```diff
@@ -15,11 +15,20 @@
 #include "PlatformDefs.h"
 
 // VC_ messages, messages can be combined
-#define OC_ERROR    0x00000001  // an error occurred, no other messages will be returned
-#define OC_BUFFER   0x00000002  // the decoder needs more data
-#define OC_OVERLAY  0x00000004  // the decoder decoded an overlay, call Decode(NULL, 0) again to parse the rest of the data
-#define OC_DONE \
-  0x00000008 // the decoder has decoded the packet, no overlay will be provided because the previous one is still valid
+enum class OCMessage
```

Comment:
OC appears throughout the code, but it's not immediately evident what it stands for outside this file. Maybe `OverlayMessage`?